includes/wikia/services/UserStatsService.class.php

```diff
@@ -1,15 +1,9 @@
 <?php
 
-class UserStatsService extends WikiaModel {
+class UserStatsService extends WikiaModel implements IDBAccessObject {
 
 	const CACHE_TTL = 86400;
-	const CACHE_VERSION = 'v1.2';
-	const USER_STATS_PROPERTIES = [
-		'editcount',
-		'editcountThisWeek',
-		'firstContributionTimestamp',
-		'lastContributionTimestamp'
-	];
+	const CACHE_VERSION = 'v2';
```

Comment:
@TK-999 — per our conversation about "cache versioning", how will recaching affect our performance and stability?